docs/build/html/_sources/getting_started.rst.txt

```diff
@@ -0,0 +1,26 @@
+Getting Started
+===================================
+
+This document explains methods that can be used to install the operator onto an existing kubernetes cluster.
+
+No matter which method you choose, the same operator will be installed.
+
+**With OLM**
+::
+operator-sdk run bundle quay.io/redhat-et-ipfs/ipfs-operator-bundle:v0.0.1 -n ipfs-operator-system
+
```

Comment:
I don't think our bundles are updated... can we remove these and document only the methods that install from sources? If people see this it is a footgun as they will end up with old code.

Reply:
Good point. Removing. We can add this back in once we can actually support using OLM.